pkg/azurefile/azurefile.go

```diff
@@ -370,17 +378,28 @@ func IsCorruptedDir(dir string) bool {
 	return pathErr != nil && mount.IsCorruptedMnt(pathErr)
 }
 
-func (d *Driver) getAccountInfo(volumeID string, secrets, context map[string]string) (rgName, accountName, accountKey, fileShareName, diskName string, err error) {
+func (d *Driver) getAccountInfo(volumeID string, secrets, reqContext map[string]string) (rgName, accountName, accountKey, fileShareName, diskName string, err error) {
 	if len(secrets) == 0 {
 		rgName, accountName, fileShareName, diskName, err = getFileShareInfo(volumeID)
 		if err == nil {
 			if rgName == "" {
 				rgName = d.cloud.ResourceGroup
 			}
-			accountKey, err = d.cloud.GetStorageAccesskey(accountName, rgName)
+			if d.cloud.KubeClient != nil {
```

Comment:
what about write a getAzureCredentials func, then it would be easy to write ut

Reply:
Since d.cloud.GetStorageAccesskey needs a fakeDriver, I will separate the getAzureCredentials func as a Driver's func in another UT related PR. Also, will change setAzureCredentials func to a Driver's func.